### ISSUE TITLE
Withdraw honey deposits for past proposals

### ISSUE THREAD

**Issue by lkngtn** [1Hive/gardens#559]
When in the deposit manager I see the following:

![image](https://github.com/1Hive/gardens/assets/4986634/5776b87b-c85d-48cc-9fe4-e778247a03be)

Many of these proposals have been cancelled/withdrawn, so I should be able to withdraw my deposit. For example: https://gardens.1hive.org/#/xdai/garden/0x8ccbeab14b5ac4a431fffc39f4bec4089020a155/proposal/111

I would expect that in the deposit manager these would all be listed as "Unlocked" and I would have the option to reclaim/withdraw them.

**Comment by kamikazebr** [comment]
Yeah i have same problem. Did you remember if that unlock automatically in the past?
